config/elasticsearch/mappings/users.json

```diff
@@ -8,6 +8,9 @@
       "type": "keyword",
       "copy_to": "search_fields"
     },
+    "badge_achievements_count": {
```

Comment:
Since we are adding new fields here for the view you will want to add these here, to the serializer, and then run a data update script to reindex all users so the new fields are present. That should all be done in a separate PR from searching.